demos/projects/NXP/mimxrt1060/main.c

```diff
@@ -527,36 +527,50 @@ int iMainRand32( void )
 }
 /*-----------------------------------------------------------*/
 
+/* *INDENT-OFF* */
```

Comment:
Line 530 and 554 are the only manual changes in the demo files, to mitigate a limitation with uncrustify: https://github.com/uncrustify/uncrustify/issues/2032